keyboards/ergotravel/via.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file also does not belong here.

Reply:
Submit a PR to https://github.com/the-via/keyboards

Reply:
working on the changes.